### ISSUE TITLE
Add an option to select the hard drive (Documents) saving format for photos

### ISSUE THREAD

**Issue by shadowpanther** [Yellow-Dog-Man/Resonite-Issues#1877]
### Is your feature request related to a problem? Please describe.

As #870 was not implemented as requested, and selects the in-game saving format instead, another option is needed.

In the end, I want to have an option to save the photo to Inventory (and spawn in-world) as WEBP, but save to my hard drive (the Documents folder) as PNG.

### Describe the solution you'd like

Add another option alongside the Encode format option to select the format the photo would be saved in when saving to the Documents folder (as opposed to when saving to the Inventory).

### Describe alternatives you've considered

None

**Comment by shadowpanther** [comment]
Keep in mind, @shiftyscales, that this request is not only for manual exporting ("Save as screenshot"), but also for automatic saving when you take the picture originally.

**Comment by shiftyscales** [comment]
Yes, I understood that, hence why I was making that clarification to the issue title, @shadowpanther.

Exporting using the "save as screenshot" context menu item uses the same mechanisms as if the photo were taken directly from a camera.

**Comment by shiftyscales** [comment]
Additional feedback for this issue was provided initially in #1878.

It highlighted the existence of the "Keep original screenshot format" setting, and its relation to this issue.

I had noted that it would be useful to consolidate that functionality into this setting, adding 'keep original format' as an option for the export format.

**Comment by jonnyawsom3** [comment]
Related to this and [Froox's comment](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/870#issuecomment-1849110594) on the previous issue, currently PNG saving is extremely slow.
I'd open a new issue for it, but PNG screenshots are already a few issues deep and I don't want to make it even more complicated...

Due to [misconfigured WebP parameters](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/874#issuecomment-2702230517), saving to inventory isn't as fast as it could be (Up to 90% faster with 1% larger files).
Then Resonite has to re-compress the WebP into a PNG, but those parameters are also misconfigured. Using [pngcheck](http://www.libpng.org/pub/png/apps/pngcheck.html) reports `zlib: deflated, 32K window, maximum compression`, which you can actually see if you check the screenshot folder after taking a quick photo. Depending on image viewer, you can watch the file being written line by line. 

Both of these config issues add up, causing a 4K screenshot to take 30 Seconds to save and 2 Minutes(!) for an 8K screenshot to finish writing to disk. That's based on file created and modified times, meaning only PNG encoding, so the still slightly sluggish WebP saving makes it even slower.

The standard for game use is usually `superfast compression` (zlib level 1), but even more conservative values like 3 should be at most 20% larger and around 80% faster. It can be even faster swapping zlib in FreeImage with [zlib-ng](https://github.com/zlib-ng/zlib-ng), which I'll open an issue for shortly.

More modern tools like [OxiPNG](https://github.com/shssoichiro/oxipng) can compress PNGs in a fraction of a second without sacrificing filesize (0.15% smaller than Resonite in under 0.8 seconds), but that would mean a more direct integration for certain formats, which I know Froox wants to avoid.
